NativeApp/src/components/Login/Input/index.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { FormInput } from 'react-native-elements';
+import { PropTypes as PT } from 'prop-types';
+import { UNOBLUE, BLACK, RED, GREY } from '../../../styles/colors';
+
+class Input extends React.Component {
```

Comment:
Not important. But you can import the Component like `import React, { Component } from 'react'`.

Then write `class Input extends Component`. Saves having to write `React.Component` all the time. :)

Reply:
done :)